README.md

```diff
@@ -24,18 +24,7 @@ See:
 
```

Comment:
Quite a few missing reference changes in here, especially around workspace package version management